### DISCUSSION TITLE
Multi-Keys/Extra-Keys [like 6k, 8k, etc]

### DISCUSSION THREAD

**Discussion by Feps-Beps** [CodenameCrew/CodenameEngine#224] [Ideas]
I don't know if the Idea like this was already posted, but here it goes:

- A Multi-Keys/Extra-Keys Feature, which allows you to make songs/charts like in VS Shaggy, VS Shaggy X Matt, etc. 
 In short - from 4k up to 9k or even more if possible

**Comment by charlesisfeline** [comment]
theres a script for it in the official discord server

**Comment by NexIsDumb** [comment]
it's already possible through scripts
there are scripts in the discord already if you wish to join
